src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer_GetDiagnosticsForSpan.cs

```diff
@@ -106,14 +106,16 @@ private class LatestDiagnosticsForSpanGetter
                 _includeSuppressedDiagnostics = includeSuppressedDiagnostics;
             }
 
-            public async Task<bool> TryGetAsync(List<DiagnosticData> list, CancellationToken cancellationToken)
+            public async Task<bool> TryGetAsync(List<DiagnosticData> list, Func<string, IDisposable> addOperationScopeOpt, CancellationToken cancellationToken)
```

Comment:
I get not wanting to annotate this entire file, but want to `#nullable enable` around this single method to avoid the Opt suffix and still get compiler validation?

Reply:
@tmat is already enabling nullable for this entire file in a separate PR that refactors most of the DiagnosticIncrementalAnalyzer code. I'll let the merge handle enabling correct nullable values for the new parameter for this method.